src/stdlib/src/ascii_string.cc

```diff
@@ -39,7 +59,7 @@ namespace perlang
     ASCIIString::~ASCIIString()
     {
         if (owned_) {
-            delete[] bytes_;
+            free((void*)bytes_);
```

Comment:
This was an oversight in the old code; it _worked_ but.. mixing `malloc` and `delete` isn't really a great idea. Either go with `malloc`+`free` or `new`+`delete` but never a combination of those. Since we are dealing with plain, C-style `char*` pointers that we then do `memcpy` and stuff using, I think C-style allocation makes a bit of sense in this case.